server/drivers/yard2LCD.c

```diff
@@ -0,0 +1,647 @@
+/***************************************************************************
+ *   Copyright (C) 2009 by M. Feser                                        *
+ *   Copyright (C) 1013 by R. Geigenberger (Update for YARD2)              *
+ *                                                                         *
```

Comment:
Please include an e-mail address where you expect to remain reachable. I think lcdproc has already too many drivers where the author can't be contacted easily.